lib/tests/tests/diagnostics/cant_abort_infallible_function.vrl

```diff
@@ -1,14 +1,14 @@
+# DIAGNOSTICS
 # result:
 #
-# error[E620]: can't abort infallible function
+# warning[E620]: can't abort infallible function
 #   ┌─ :2:1
 #   │
 # 2 │ now!()
-#   │ ^^^- remove this abort-instruction
+#   │ ^^^- remove this abort (!) instruction
 #   │ │
 #   │ this function can't fail
 #   │
-#   = see documentation about error handling at https://errors.vrl.dev/#handling
```

Comment:
Are you sure you want to drop this link? I think it is useful to show users how to handle this warning.